### PR TITLE
[hi] Add content/hi/docs/setup/_index.md

### PR DESCRIPTION
**Pull request by keshavcodex**
This PR is a part of ongoing hi localization attempt here #29353

/language hi

### CODE REVIEW

**Comment by keshavcodex** [comment]
I signed it

**Review by Arhell — COMMENTED** [review]
/check-cla

**Comment by Arhell** [comment]
@keshavcodex  
first, you need to confirm CLA 
second, here are the members of the organization who are responsible for translation into hindi
https://github.com/kubernetes/website/blob/main/OWNERS_ALIASES#L78

**Comment by keshavcodex** [comment]
> @keshavcodex
> first, you need to confirm CLA
> second, here are the members of the organization who are responsible for translation into hindi
> https://github.com/kubernetes/website/blob/main/OWNERS_ALIASES#L78

i have already confirmed the CLA, still bot is stopping me. but I have asked in support,i hope they will revert back soon.

**Comment by ShivamTyagi12345** [comment]
Hey @keshavcodex  ,Please edit the PR name to the format `Add <file-location>`

**Comment by keshavcodex** [comment]
> Hey @keshavcodex ,Please edit the PR name to the format `Add <file-location>`

ok sure

File: content/hi/docs/setup/_index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## {{% heading "whatsnext" %}}
+
+- [कुबेरनेट्स डाउनलोड करें](/releases/download/)
+- डाउनलोड और [उपकरण स्थापित करें](/docs/tasks/tools/) `kubectl` सहित।
+- अपने नए क्लस्टर के लिए [कंटेनर रनटाइम](/docs/setup/production-environment/container-runtimes/) चुनें
+- क्लस्टर सेटअप के लिए [सर्वोत्तम प्रक्रियाओं](/docs/setup/best-practices/) के बारे में जानें
+
+
+कुबेरनेट्स को इसके {{< glossary_tooltip term_id="control-plane" text="control plane" >}} के लिए Linux पर चलाने के लिए डिज़ाइन किया गया है। अपने क्लस्टर के भीतर आप लिनक्स या अन्य ऑपरेटिंग सिस्टम पर एप्लिकेशन चला सकते हैं, जिसमें शामिल हैं विंडोज़।
```
Comment (sftim) [inline content/hi/docs/setup/_index.md:59]: nit: I would transcribe or translate the text in `text="control plane"`

Comment (keshavcodex) [reply content/hi/docs/setup/_index.md:59]: oops, i have lost this pull request branch from my computer so should i leave it as it is or i should create new PR and then will close this PR.

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add website/tree/main/content/hi/docs/setup/_index.md

File: content/hi/docs/setup/_index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+
+
+
+
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/_index.md:5]: These lines are not required

File: content/hi/docs/setup/_index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+content_type: concept
+no_list: true
+card:
+  name: setup
+  weight: 20
+  anchors:
+  - anchor: "#सीखने का माहौल"
+    title: सीखने का माहौल
+  - anchor: "#उत्पादन वातावरण"
+    title: उत्पादन वातावरण
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/_index.md:18]: I would prefer `उत्पादन` -> `प्रोडक्शन` here

File: content/hi/docs/setup/_index.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+<!-- overview -->
+
+यह भाग कुबेरनेट्स को स्थापित करने और चलाने के विभिन्न तरीकों को सूचीबद्ध करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/setup/_index.md:23]: `स्थापित` -> `सेटअप` ?

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/setup/_index.md (hunk 1 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/setup/_index.md:59]: You have it on GitHub. Right?

Comment (keshavcodex) [reply content/hi/docs/setup/_index.md:59]: ok @vedant-kakde

**Comment by keshavcodex** [comment]
I had opened it [Here](https://github.com/kubernetes/website/pull/29514) because i had missed this branch in local computer, i apologise for the inconvenience.

**Pull request closed without merging**